CMakeLists.txt

```diff
@@ -7,6 +7,27 @@ set(SRC "${CMAKE_CURRENT_SOURCE_DIR}")
 set(CMAKE_POSITION_INDEPENDENT_CODE ON) # This does not set -fPIC :(
 set(CMAKE_CXX_FLAGS "-fPIC")
 
+## ZSerio binary
+####
+option(ZSERIO_JAR_PATH "Path to zserio jar.")
+if (ZSERIO_JAR_PATH)
+  set(ZSERIO "java")
+  list(APPEND ZSERIO "-jar")
+  list(APPEND ZSERIO "${ZSERIO_JAR_PATH}")
+
+  message("Using zserio jar: ${ZSERIO_JAR_PATH}")
+else()
+  find_program(ZSERIO zserio)
```

Comment:
Consider dropping support for a zserio executable completely as this is not the way zserio deployed/published.